plugins/builds/update.js

```diff
@@ -195,8 +195,11 @@ module.exports = config => ({
                                     buildLink: `${buildFactory.uiUri}/pipelines/${pipeline.id}/builds/${id}`
                                 });
 
+                                const skipFurther = /\[(skip further)\]/.test(newEvent.causeMessage);
```

Comment:
what is this?

Reply:
From the list view you can build a single job; having "skip further" in the message indicates that it shouldn't build any jobs that are after this job in the pipeline.

Reply:
is this a part of https://github.com/screwdriver-cd/screwdriver/issues/1967? just wondering if it's documented somewhere

Reply:
I don't think its documented anywhere; it was just something I talked with Jithin about when planning what the 3 buttons in each row of the list view should do

Reply:
@InderH we need to document this. Please update guide with this info.

Reply:
@jithine Which guide are you referring to?

Reply:
At docs.screwdriver.cd. Possibly an FAQ section. Not sure where else it fits.